2017/Dockerfile

```diff
@@ -36,12 +36,13 @@ RUN chmod 755 /usr/local/bin/start.sh
 
 # Setup for reading Health Insurance ID Card
 
-# Install script for plugin of Health Insurance ID Card will use killall command
-COPY health_insurance_card /tmp/health_insurance_card/
+ADD health_insurance_card/mLNHIICC_Setup.Ubuntu16.tar.gz /tmp/health_insurance_card/
```

Comment:
I've fixed this problem in https://github.com/chihchun/personal-income-tax-docker/pull/1/commits/aa4cb1794379cec35c5649a01d4787bb2add2a19, please have a look. Thanks.